tests/utilities/test_wheel.py

```diff
@@ -69,6 +69,7 @@ def array_teardown():
     tiledb.Array.delete_array(_FULL_URI, ctx=ctx)
 
 
+@pytest.mark.skip(reason="Race condition in tests.")
```

Comment:
These wheel tests fail ~20% of the time. They can't be xfailed, they must be skipped because the bug is in the `array_teardown` fixture.